mcache/tests/test_check.py

```diff
@@ -1,18 +1,19 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+
 import subprocess
 import os
 import time
-
-from datadog_checks.mcache import Memcache
 import pytest
 import bmemcached
 from bmemcached.exceptions import MemcachedException
 
-from .common import PORT, SERVICE_CHECK, HOST, USERNAME, PASSWORD
+from datadog_checks.utils.platform import Platform
+from datadog_checks.mcache import Memcache
 
-HERE = os.path.abspath(os.path.dirname(__file__))
+from common import (HERE, PORT, SERVICE_CHECK, HOST, USERNAME, PASSWORD, DOCKER_SOCKET_DIR, DOCKER_SOCKET_PATH,
+                    UNIXSOCKET_DIR, UNIXSOCKET_PATH)
 
 GAUGES = [
```

Comment:
Same for the metrics, can they be put in their own file or in `common.py` maybe ?